### PR TITLE
Adding Update and Delete Mentors API

### PR DESCRIPTION
**Pull request by Keilvin**

### CODE REVIEW

**Review by Naton-1 — APPROVED** [review]

File: utils.py [hunk 1]
```diff
@@ -236,49 +276,14 @@ def store_mentor_question(type, question_text, conn, cursor, mc_options):
         return False
 
 def modify_mentor_question(question_id, question_text, conn, cursor):
-    query = "UPDATE question SET question = %s WHERE questionID = %s"
+    query = "UPDATE `question` SET `question` = %s WHERE `questionID` = %s"
```
Comment (JustinRMoy) [inline utils.py:279]: the ` around question and questionID are unnecessary.

File: resources/mentors.py [hunk 2]
```diff
@@ -246,21 +246,47 @@ def post(self):
...
 
-            raise ReturnSuccess(mentorQuestions, 200)
+                delete_query = "INSERT INTO `deleted_question` (`questionID`, `audioID`, `imageID`, `type`, `questionText`) VALUES (%s, %s, %s, %s, %s)"
+                postToDB(delete_query, (question_data[0][0], question_data[0][1], question_data[0][2], question_data[0][3], question_data[0][4]))
+
+                la_query = "SELECT `logID` FROM `logged_answer` WHERE `questionID` = %s"
+                la_results = getFromDB(la_query, question_data[0][0], conn, cursor)
+
+                for log in la_results:
+                    log_query = "UPDATE `logged_answer` SET `questionID` = %s, `deleted_questionID` = %s WHERE `logID` = %s"
```
Comment (JustinRMoy) [inline resources/mentors.py:278]: we're not using logged_answer. The student answers to mentor questions go to the mentor_responses table

File: __init__.py [hunk 3]
```diff
@@ -171,6 +171,10 @@ def index():
 api.add_resource(MentorPreference, API_ENDPOINT_PREFIX + 'mentorpreference')
 api.add_resource(StudentResponses, API_ENDPOINT_PREFIX + 'studentresponses')
 api.add_resource(MentorQuestions, API_ENDPOINT_PREFIX + 'mentorquestions')
+api.add_resource(ModifyMentorQuestions, API_ENDPOINT_PREFIX + 'modifymentorquestions')
+api.add_resource(DeleteMentorQuestion, API_ENDPOINT_PREFIX + 'deletementorquestions')
```
Comment (JustinRMoy) [inline __init__.py:175]: You need to import the classes for these two API endpoints at line 33

**Review by JustinRMoy — CHANGES_REQUESTED** [review]
I made 3 comments on things that need to be changed before we merge. 1 is critical as it will just result in an error that being the import statements for the api

File: resources/mentors.py (hunk 2 above)
Comment (Keilvin) [reply resources/mentors.py:278]: logged answer does not in fact log the answers, log answer logs the session statistics and the log time as well as when the questions are added and deleted. In this case it updates the log of this particular questionID to be a deleted question.

**Review by JustinRMoy — APPROVED** [review]

**Pull request merged**